File: split/eqworddefine.pas
procedure eqworddefine(p: halfword; w: integer);
begin
  if xeqlevel[p] <> curlevel then
  begin
    eqsave(p, xeqlevel[p]);
    xeqlevel[p] := curlevel;
  end;
  eqtb[p].int := w;
end;

File: cpp/eqworddefine.cpp
#include "eqworddefine.h"
#include "eqsave.h"

void eqworddefine(halfword p, int w)
{
	if (xeqlevel[p] != curlevel)
	{
		eqsave(p, xeqlevel[p]);
		xeqlevel[p] = curlevel;
	}
	eqtb[p].int_ = w;
}
